transforms/helpers/validation-helper.js

```diff
@@ -1,5 +1,25 @@
+const minimatch = require("minimatch");
```

Comment:
It looks like this dependency didn't make it into `package.json`

Reply:
How did it work? 🤔 Let me check the `package.lock`

Reply:
Ahh! Its added as dependency for `babel-core`. Let me update the `package.json/lock`